ContextualPopupDecorator/tests/ContextualPopupDecorator-specs.js

```diff
@@ -1,78 +1,100 @@
-import {mount} from 'enzyme';
 import {FloatingLayerDecorator} from '@enact/ui/FloatingLayer';
+import '@testing-library/jest-dom';
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
 import {ContextualPopupDecorator} from '../ContextualPopupDecorator';
 import Button from '../../Button';
 
 const ContextualButton = ContextualPopupDecorator(Button);
 
 describe('ContextualPopupDecorator Specs', () => {
-	test('should bind callback to instance', () => {
```

Comment:
This test was removed because Testing Library does not provide(by choice) access to the component instance.
"It is intended to test the app as a user would experience it."